build-package.sh

```diff
@@ -28,6 +28,9 @@ case "$1" in
         ;;
     stage)
         craftctl default
+        if [ -e ${CRAFT_PROJECT_DIR}/test-debs ]; then
+            /bin/cp -a ${CRAFT_PROJECT_DIR}/test-debs/* ${CRAFT_STAGE}/local-debs/
```

Comment:
Why the full path to cp?

Reply:
Ok, my fault... that error was in another place. Fixed.